relay-kafka/Cargo.toml

```
@@ -21,7 +21,7 @@ rmp-serde = { workspace = true, optional = true }
 serde = { workspace = true }
 serde_json = { workspace = true, optional = true }
 thiserror = { workspace = true }
-sentry-kafka-schemas = { workspace = true, default_features = false, optional = true }
+sentry-kafka-schemas = { workspace = true, optional = true , features = ["type_generation"]}
```

```suggestion
sentry-kafka-schemas = { workspace = true, optional = true }
```

1. Why do we need this `type_generation` feature now?
2. The feature is already enabled by default (so removing it should work)

---

1. `type_generation` is needed to get `schema_types.rs` generated and included (https://github.com/getsentry/sentry-kafka-schemas/blob/20b901afde1b40f6b461515e904084461a595c52/rust/lib.rs#L10) so we can use this `PROTOS` constant (https://github.com/getsentry/sentry-kafka-schemas/blob/20b901afde1b40f6b461515e904084461a595c52/rust/build.rs#L190) when we validate the schema of the `scuba-items` topic since it's now part of the Relay config.
2. Indeed, I think I wanted it to show we relied on it now. I removed it.

---

Since we fixed the crate (https://github.com/getsentry/sentry-kafka-schemas/pull/407), I disabled the default features again.